cvat/apps/dataset_manager/task.py

```diff
@@ -103,25 +110,17 @@ def add_prefetch_info(cls, queryset):
             'segment__task__project__owner',
             'segment__task__project__assignee',
```

Comment:
@zhiltsov-max, @Eldies,
Is this information really used in cases except for export?
```
'segment__task__owner',
'segment__task__assignee',
'segment__task__project__owner',
'segment__task__project__assignee'
```

Reply:
```
'segment__task__project__owner',
'segment__task__project__assignee'
```
I also couldn't find where these 2 could be used.